assembl/static2/workspaces/assembl-test-editor-utils/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "assembl-test-editor-utils",
+  "version": "1.0.0",
```

Comment:
Not to be a party pooper, as these are plugins created by you, under the Bluenove domain. As they are under Bluenove, they should likely follow the semver that we use, which recommends 0.1.0 as a start point, as it's the first dev done. We can iterate more and more until long-term requirements are satisfied, at which point, the plugins can become 1.0.0. I know we'll be putting this on prod soon, so I'm flexible on this..

https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
done!